js/app/util.js

```diff
@@ -951,13 +951,10 @@ define([
      * @param text
      * @returns {String}
      */
-    let unicodeToString = (text) => {
-      let result = text.replace(/\\u[\dA-F]{4}/gi,
-        function (match) {
-          return String.fromCharCode(parseInt(match.replace(/\\u/g, ''), 16));
-      });
-      return result.substring(0, 2) == "u'" ? result.substring(2, result.length - 1) : result;
-    };
+    const unicodeToString = (text) => {
+        const result = text.replace(/\\u[\dA-F]{4}/gi, (match) => String.fromCharCode(parseInt(match.replace(/\\u/g, ''), 16)))
+        return result.substring(0, 2) == "u'" ? result.substring(2, result.length - 1) : result;
```

Comment:
You can remove that `;` at the end. You don't need them. I made a mistake when I copied it over from his original code!

Reply:
done ✝